CHANGELOG.rst

```diff
@@ -17,6 +17,19 @@ Unreleased
 * Switch from ``edx-sphinx-theme`` to ``sphinx-book-theme`` since the former is
   deprecated
 
+[4.0.0] - 2023-05-02
+********************
+Changed
+* Implement openedx-events Event Bus Consumer API
+* **BREAKING CHANGE**: Remove ``consume_events`` management command as this
+  command will be provided by openedx_events. To replay events using the
+  openedx-events version of the management command, pass ``--extra
+  '{"offset_time": "2023-01-08T06:46:22"}'`` instead of ``-o
+  2023-01-08T06:46:22``.
+* **BREAKING Change**: ``offset_timestamp`` argument has been removed from
+  ``consume_indefinitely`` method. It is now added as an optional argument
+  named ``offset_time`` to ``KafkaEventConsumer`` constructor.
```

Comment:
Some changes:

- Remove hard wraps (it confuses the rST parser when lists are wrapped, I think?)
- Move theme item from Unreleased section into 4.0.0 section
- Mention `reset_offsets_and_sleep_indefinitely` as another changed method
- Update date (anticipating merge tomorrow)

```suggestion
[4.0.0] - 2023-05-05
********************
Changed
=======
* Implement openedx-events Event Bus Consumer API
* **BREAKING CHANGE**: Remove ``consume_events`` management command as this command will be provided by openedx_events. To replay events using the openedx-events version of the management command, pass ``--extra '{"offset_time": "2023-01-08T06:46:22"}'`` instead of ``-o 2023-01-08T06:46:22``.
* **BREAKING Change**: ``offset_timestamp`` argument has been removed from the ``consume_indefinitely`` and ``reset_offsets_and_sleep_indefinitely`` methods. It is now added as an optional argument named ``offset_time`` to ``KafkaEventConsumer`` constructor.
* Switch from ``edx-sphinx-theme`` to ``sphinx-book-theme`` since the former is deprecated
```

Reply:
Thanks! Updated.